src/corehost/packaging/dir.props

```diff
@@ -23,7 +23,7 @@
 
   <PropertyGroup>
     <PackageOutputPath>$(PackagesOutDir)</PackageOutputPath>
-    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\1.0.1-rc2-23712\runtime.json</RuntimeIdGraphDefinitionFile>
+    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\1.0.1-rc2-23931\runtime.json</RuntimeIdGraphDefinitionFile>
```

Comment:
> Does your UpdateDependencies script have a way to update non-project.json files?

Yes it does. It is just a C# .NET Core app, so it can do anything to the repo it needs to.

Here is the update:  https://github.com/dotnet/cli/pull/2268/commits/e0542adf51fed1554cb660fceaffcadfabc17438